datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/DurationDeserializer.java

```diff
@@ -49,6 +53,11 @@
 
     public static final DurationDeserializer INSTANCE = new DurationDeserializer();
 
+    /**
+     * Since 2.12
+     */
+    private DurationPattern pattern;
```

Comment:
Recommend adding some JavaDoc here to describe the new variable. Also suggest naming it `_durationPattern` for consistency with other private vars and to be a little more specific.

Reply:
Done